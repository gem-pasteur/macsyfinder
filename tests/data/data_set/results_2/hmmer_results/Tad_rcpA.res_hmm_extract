# gene: Tad_rcpA extract from results_2/hmmer_results/Tad_rcpA.search_hmm.out hmm output
# profile length= 225
# i_evalue threshold= 0.001
# coverage threshold= 0.500
# hit_id replicon_name position_hit hit_sequence_length gene_name gene_system i_eval score profile_coverage sequence_coverage begin end
GCF_000006845_001030	GCF_000006845	103	723	Tad_rcpA	5.400e-34	114.300	0.996	0.333	475	715
