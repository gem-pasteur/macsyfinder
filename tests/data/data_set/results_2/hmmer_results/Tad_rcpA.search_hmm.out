# hmmsearch :: search profile(s) against a sequence database
# HMMER 3.1b2 (February 2015); http://hmmer.org/
# Copyright (C) 2015 Howard Hughes Medical Institute.
# Freely distributed under the GNU General Public License (GPLv3).
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# query HMM file:                  tests/data/data_set/models/set_1/profiles/Tad_rcpA.hmm
# target sequence database:        tests/data/base/one_replicon.fasta
# output directed to file:         results_2/hmmer_results/Tad_rcpA.search_hmm.out
# sequence reporting threshold:    E-value <= 0.1
# number of worker threads:        0
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Query:       Tad_rcpA  [M=225]
Scores for complete sequences (score includes all domains):
   --- full sequence ---   --- best 1 domain ---    -#dom-
    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description
    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------
    5.4e-34  114.3   1.6    5.4e-34  114.3   1.6    1.9  2  GCF_000006845_001030  


Domain annotation for each sequence (and alignments):
>> GCF_000006845_001030  
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ?   -0.8   0.0     0.043        81       8      74 ..     243     330 ..     239     335 .. 0.53
   2 !  114.3   1.6   2.8e-37   5.4e-34       1     224 [.     475     715 ..     475     716 .. 0.88

  Alignments for each domain:
  == domain 1  score: -0.8 bits;  conditional E-value: 0.043
              Tad_rcpA   8 kvlkqlGvslsavig.gvngslansatgsgF....................gtaekiesgaleigakklkallnaleekglvkvLaep 74 
                            +lk l  +++ +i+ + n +l n + + g+                     ++ + ++  l+++  ++ ++l++l +++ +++ a  
  GCF_000006845_001030 243 VTLKRLNNDTQLIITtTGNWELVNKSAAPGYftfqvlpkkqnlesggvnnaPKTFTGRKISLDFQDVEIRTILQILAKESGMNIVASD 330
                           5555555555555543333344444444444455555555555544432221122223444566666799999999999999988865 PP

  == domain 2  score: 114.3 bits;  conditional E-value: 2.8e-37
              Tad_rcpA   1 rvvevsrkvlkqlGvslsavig...gvngslansatgsgF................gtaekiesgaleigakklkallnaleekglvkv 70 
                           r+ve    ++ +lGv++ a+     + ++s+ + +++sgF                  a++  s + +i++  l+  l a e+ + +k+
  GCF_000006845_001030 475 RIVEAADGFSRDLGVKFGATGRkklKNETSAFGWGVNSGFgggdkweaqtkinlpvAAAANSISLVRAISSGALNLELSASESLSKTKT 563
                           68999999********999954555555556666666677888888888777666544667778888889999**************** PP

              Tad_rcpA  71 LaepnLvalsgesAkflvGGelpiptgnetn..gsvtieykeyGvkLnltptvlsngrisLkvstevSeldsegsvslgggstvpslrt 157
                           La+p + + + ++Ak+ +G e+p+ +++     +s+++e k+  ++L++tp +   g+i ++v+++  +++   +   g+  t+  ++t
  GCF_000006845_001030 564 LANPRVLTQNRKEAKIESGYEIPFTVTTASGggNSTNTELKKAVLGLTVTPNITPDGQIIMTVKINKDSPA---QCASGN-NTILCIST 648
                           ************************99988777788999***************************998888...444455.8******* PP

              Tad_rcpA 158 rrvettvelasgeslvlaGllsseekesidklPllgdiPilGalfrsketlknqteLviivtpelvk 224
                           +  +t+ ++++g++l+++G+ ++++ ++++k+PllgdiP++G+lf+ +  + + +eL+i++tp++++
  GCF_000006845_001030 649 KSLNTQAMVENGGTLIVGGIYEENNGNTLTKVPLLGDIPVIGNLFKTRGKKTDRRELLIFITPRIID 715
                           *****************************************************************97 PP



Internal pipeline statistics summary:
-------------------------------------
Query model(s):                              1  (225 nodes)
Target sequences:                         1886  (549846 residues searched)
Passed MSV filter:                        45  (0.02386); expected 37.7 (0.02)
Passed bias filter:                       37  (0.0196182); expected 37.7 (0.02)
Passed Vit filter:                         4  (0.00212089); expected 1.9 (0.001)
Passed Fwd filter:                         1  (0.000530223); expected 0.0 (1e-05)
Initial search space (Z):               1886  [actual number of targets]
Domain search space  (domZ):               1  [number of targets reported over threshold]
# CPU time: 0.01u 0.00s 00:00:00.01 Elapsed: 00:00:00.01
# Mc/sec: 12371.53
//
[ok]
